<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="musicButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>238</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Umpush</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ZENE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="optionsButton">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>122</y>
      <width>238</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Umpush</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BEÁLLÍTÁSOK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="radarButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>122</y>
      <width>238</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Umpush</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TOLATÓRADAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataButton">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>0</y>
      <width>238</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>JÁRMŰ</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
